functionAsArgument.js

```diff
@@ -0,0 +1,8 @@
+function f1(name){
+    return "heyy  "+name;
+}
+function f2(func){
+    console.log(func);
+}
+
+f2(f1("prasad"));
```

Comment:
Can you please add expected output as an inline comment at line 8. Also please add question in the form of comments at the line 1.

Reply:
sure